TestsExample/src/Test1097.tsx

```diff
@@ -50,6 +50,7 @@ function First({ navigation }: NativeStackScreenProps<ParamListBase>) {
     hideNavigationBar: false,
     autoCapitalize: 'sentences',
     placeholder: 'Some text',
+    placement: 'stacked',
```

Comment:
I would not change this test, as it was related to the other topic (related to Windows support, I guess? See #1097).
Also, if 'stacked' value is default, do we require defining this property here?

Reply:
`Test1097` become test case for all search bar related stuff a long time ago -- that's why there is so much scrollview / searchbar code. Also we should not rename it as many PRs refer to this test in their description.

> Also, if 'stacked' value is default, do we require defining this property here?

It's easier to see where the value of prop should be placed / changed when looking for usage example / testing.